### ISSUE TITLE
Users are not able to access “Size” link present in “Installation” section via keyboard.: A11y_VisualStudioCodeClient_UserWantsToExplore Customizations and Adding extension_UserWantsToReading details on specific extension

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#285486]
Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Feng.ms%2Fdocs%2Fcoreai%2Fdevdiv%2Fone-engineering-system-1es%2F1es-docs%2Fes-chat%2Findex&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221175207%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=ju8Raw6mIhT5G3gWbZSkS0CAN2zkv2ycA5ifSSqoxOw%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fchannel%2F19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2FCAI%2520Accessibility%2520Support%2520Channel%3FgroupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221198042%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=DCNVsZe8oDZWqm97AI0ee4OoWWecR6zw6qlhTuzP7MI%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Foutlook.office.com%2Fbook%2Fmicrosoft2779007%2540microsoft.onmicrosoft.com%2Fs%2FbLeGBltMW0qI0WRLsTqx2A2%3Fismsaljsauthenabled&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221214666%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=CnCCdDgEmll2oUQUdMjXusAgV8PneqZVYvzgxAfk7Ik%3D&reserved=0). 

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."  

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".** 

### GitHub Tags:  

#A11yTCS;#Visual Studio Code Client;#E2E_VisualStudioCodeClient_Win32_Dec2025;#DesktopApp;#FTP;#Win32;#A11ySev2;#A11yMAS;#WCAG2.1.1; #Keyboard; 

### Environmental Details:  

App Name: Visual Studio Code Client  
#URL: Visual Studio Code Insider 
Microsoft Windows 11 Enterprise  
OS Build:  26200.7392 

### Repro Steps:  

1.Open Visual Studio Code Insiders editor 
2.Using "Ctrl+Shift+X" key navigate to the extension side view bar option. 
3.Using Tab key navigate to the "More" option and select different options ("Show installed Extension, Show outdated extension, Show popular Extension,") 
4.Click on any installed extension 
5.Try to read all the details about specific extension 
6. Observe the issue whether Users are able to access “Size” link present in “Installation” section via keyboard or not.

### Actual Results:  

Users are not able to access “Size” link present in “Installation” section via keyboard.  

### Expected Results:  

Users should be able to access “size” link present in “Installation” section via keyboard  

### User Impact:  

Keyboard only users will miss “Size” functionality present in “Installation” section.

https://github.com/user-attachments/assets/df2a6f35-0e1e-40b0-b12a-b534ed76886b

**Comment by mjbvz** [comment]
@meganrogge I unassigned myself because this part of the UI is owned by @sandy081
